src/Symfony/Component/Console/Command/Command.php

```diff
@@ -160,7 +160,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
     }
 
     /**
-     * Initializes the command just after the input has been validated.
+     * Initializes the command before the input is validated.
```

Comment:
Actually most part of the input validation happens in `Input::bind()` before that  `initialize()` is called, `Input:: validate()`. I would reword it relying on method names